src/tag.h

```diff
@@ -22,7 +22,7 @@ class HSTag : public Object {
     Attribute_<bool>         floating = {"floating", false};
     Attribute_<std::string>  name = {"name", {}};   // name of this tag
     int             flags;
-    struct HSStack* stack;
+    std::shared_ptr<HSStack> stack;
```

Comment:
Why is stack a pointer and not a simple member object? For the tree interface?

Reply:
Yes, it can simply be a member of HSTag